### DISCUSSION TITLE
Solution for Issues with SSL is not clear

### DISCUSSION THREAD

**Discussion by michkowalczuk** [AliDehbansiahkarbon/ChatGPTWizard#18] [General]
Your solution for SSL problem is not clear.
Where can I find SSL libraries you mentioned?

Second approach with open project does not work for me.

**Comment by AliDehbansiahkarbon** [comment]
SSL libraries are in the Resource folder of this project.
I just made a big change about the Http client, now it will try with built in Http client and doesn't need SSL libraries for the first attempt, but sometimes the response in this way is not a valid json for some reasons and in this case it will try with Indy in second attempt that needs SSL libraries.

I hope it helps you.
Pull the latest source code and give it a go please.
@michkowalczuk
